pkg/webhook/fields.go

```diff
@@ -91,6 +92,21 @@ func ConfigSyncMetadata(set *fieldpath.Set) *fieldpath.Set {
 	return csSet
 }
 
+// OnlyNotifiedAnnotation returns if the filePath set only includes the notified annotation.
```

Comment:
This is not needed anymore. It was used to prevent the fight between the admission webhook and the notification controller. After Karl made the change in the admission webhook (https://github.com/GoogleContainerTools/kpt-config-sync/pull/242/files#diff-9e60d0c27478b517b75a5e3e5dbd109c91c99abe0afc21b256ee8f9827d7502cR100) and our discussion, we decided to not control it at this level.

Reply:
done